code-challenges.js

```diff
@@ -8,62 +8,240 @@
 
 // Pseudo code:
 
+// 1.) create 2 parameters for the arguments for the arguments to return to.
+// 2.) Build a else/ if statement brings the action of determining which string has the most charactere
+    // * 
+
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 // Expected outcome: "cherry"
 
+// const fruitFoot = (string1, string2) => {
+//     if (string1.length > string2.length) {
+//         return string1
+//     } else {  
+//         return string2
+// }
+// }
+
+//     console.log(fruitFoot(fruit1, fruit2)) 
+//     console.log(fruitFoot(fruit3, fruit4))
+
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
 
-const temperature1 = 42
-// Expected output: "42 is below boiling point"
+//              key:
+                
+//              * = example
+
+
+// 1.) We have to give a value for the code to compare the given temperature to the boiling point.
+
+//      *const temp = (point1)
+
+
+// 2.) We set the conditions (else/if statements) for the value for the comparisons to take place for each temperature.
+
+//      *const temp = (point1) => {
+//          if ( point1 < 212)
+//          }
+//     else if (point1 > 212) {
+//  }
+//   else if (point1 = 212) {
+//  }
 
-const temperature2 = 350
-// Expected output: "350 is above boiling point"
 
-const temperature3 = 212
+// 3.) We make functions call (console.log) that will take the values (in a form of an argument) for the terminal; to run the code for each temperature. Comparing it to the boiling point.
+
+// *   console.log(temp(temperature1))
+//     console.log(temp(temperature2))
+//     console.log(temp(temperature3))
+
+
+// 4.) Give a return of the answer and write out each string that coresponds with the number that is greater than, less than, or equal to the boiling point.
```

Comment:
The final function on line 70 is great. However, your pseudo code is all over the place. I suggest removing all the examples of what you're talking about and instead have clear and concise lines describing each step your code is taking. You can even put your pseudo code with your function and go line by line, that way you can have your pseudo code right by the code it is describing.